tests/test_run.py

```diff
@@ -142,3 +142,9 @@ def make_env():
         train_freq=5,
     )
     model.learn(total_timesteps=150)
+
+
+@pytest.mark.parametrize("normalize_advantage", [False, True])
+def test_advantage_normalization(model_class, normalize_advantage):
+    model = MaskablePPO("MlpPolicy", "CartPole-v1", n_steps=64, normalize_advantage=normalize_advantage)
```

Comment:
not sure if it works with CartPole, let's see...

Reply:
Hey you may need to approve the workflow run ;)

Reply:
i did and there is some failures already ;)